src/main/lrsql/admin/protocol.clj

```diff
@@ -9,7 +9,7 @@
     "Authenticate by looking up if the account exists in the account table.")
   (-existing-account? [this account-id]
     "Check that the account with the given ID exists in the account table. Returns a boolean.")
-  (-delete-account [this account-id]
+  (-delete-account [this account-id oidc-enabled?]
     "Delete the account and all associated creds. Assumes the account has already been authenticated.")
```

Comment:
Should update the docstring with the extra arg.

Reply:
https://github.com/yetanalytics/lrsql/pull/294/commits/f1a6bbf81a88618fb891c40655bc56aee7498345